op-chain-ops/genesis/config.go

```diff
@@ -72,6 +72,9 @@ type DeployConfig struct {
 	// Seconds after genesis block that Regolith hard fork activates. 0 to activate at genesis. Nil to disable regolith
 	L2GenesisRegolithTimeOffset *hexutil.Uint64 `json:"l2GenesisRegolithTimeOffset,omitempty"`
 
+	// Configurable extradata. Default to []byte("BEDROCK") if empty.
+	L2GenesisBlockExtraData []byte `json:"l2GenesisBlockExtraData,omitempty"`
```

Comment:
fwiw, none of the others use `omitempty` so i think its fine to leave that out

Reply:
All except L2GenesisRegolithTimeOffset which I guess I was copying.  Let me know which you prefer? if I get rid of it here I can also get rid of it for L2GenesisRegolithTimeOffset for consistency I guess. Unless there's some good reason it should be different.

Reply:
Ah, I feel like it was added there because its a pointer and `nil` can mean something different than `0`. I believe `nil` means never and `0` means now

Reply:
I prefer leaving out `omitempty` in the case of `L2GenesisBlockExtraData`, curious what you think

Reply:
removed omitempty here, but left it with l2genesisregolithtimeoffset, since that one seems like it will in general be unspecified.

Reply:
Confirming your understanding of `L2GenesisRegolithTimeOffset` is correct - not setting it is a valid option (never) and different to 0 (starting from bedrock block).